kedro-datasets/README.md

```diff
@@ -17,6 +17,30 @@ Welcome to `kedro_datasets`, the home of Kedro's data connectors. Here you will
 pip install kedro-datasets
 ```
 
+### Install dependencies at a group-level
+
+Data types are broken into groups e.g. `pandas`, `spark` and `pickle`. Each group has a collection of data types e.g.`pandas.CSVDataset`, `pandas.ParquetDataset` and more. You can install dependencies for an entire group of dependencies as follows:
```

Comment:
I do find the term "data types" a bit weird, when we call them "datasets" everywhere else.

Reply:
Yes let's call them datasets.

```suggestion
Datasets are organised into groups e.g. `pandas`, `spark` and `pickle`. Each group has a collection of datasets, e.g.`pandas.CSVDataset`, `pandas.ParquetDataset` and more. You can install dependencies for an entire group of dependencies as follows:
```

?